axelor-supplychain/src/main/resources/domains/MrpLine.xml

```diff
@@ -19,7 +19,8 @@
   		<string name="relatedToSelectName" title="Related to"/>
       	<one-to-many name="mrpLineOriginList" ref="com.axelor.apps.supplychain.db.MrpLineOrigin" mappedBy="mrpLine" title="Origins"/>
   		<boolean name="proposalGenerated" default="false" required="true" readonly="true"/>
-  		<string name="proposalSelect" selection="mrp.line.proposal.ref.select" readonly="true"/>
+  		<boolean name="toProcess" title="To process" default="false"/>
```

Comment:
Please rename `toProcess` to `proposalToProcess` (the title can stay the same), to avoid confusion.